### ISSUE TITLE
Error in console on playfield load

### ISSUE THREAD

**Issue by AlexS-empyrion** [RavienGaming/Reforged-Eden-2#327]
### Reforged Eden 2 Build Number

22

### Game mode

Survival

### Tested in vanilla

Issue does not happen in vanilla

### Description of issue

Every playfield load have this error in console:

26-06:33:00.566 09_33 -ERR- Error copying C:\Program Files (x86)\Steam\steamapps\common\Empyrion - Galactic Survival/Content/Scenarios/Reforged Eden 2/Playfields/SpaceWarpTarget/playfield.yaml: Could not find file 'C:\Program Files (x86)\Steam\steamapps\common\Empyrion - Galactic Survival\Content\Scenarios\Reforged Eden 2\Playfields\SpaceWarpTarget\playfield.yaml'.

I don't know if it affected game in any way. Just reporting.

**Comment by RavienGaming** [comment]
No idea why that error pops up but I haven't noticed any issues either.

This is likely a game bug and will need to be reproduced and reported using a vanilla save game.
When reproducing a game bug in a vanilla save, make sure you are using the same mode _(singleplayer/co-op/dedicated multiplayer, survival/creative)_ as what you were using when the issue happened in the scenario.
To see a list of known game bugs or to report a new one please see: https://empyriononline.com/forums/bugs.24/

**Comment by AlexS-empyrion** [comment]
I've checked Vanilla survival. It doesn't appears there.  And there is also no such file in space warp target folder of Vanilla.
So it may be RE2 specific.

**Comment by RavienGaming** [comment]
No, it would be a game bug either way. We cannot control what file the game is looking for, nothing I can do to fix it.
